components/RecipeForm.js

```diff
@@ -33,27 +32,36 @@ export default function RecipeForm({ onSubmit, onDelete, data }) {
           },
         ]
   );
+
   const router = useRouter();
+  const [selectedTags, setSelectedTags] = useState(data ? data.tags : []);
+
+  function handleTagChange(label) {
+    const isAlreadySelected = selectedTags.includes(label);
+    let newTags = label;
+    if (isAlreadySelected) {
+      newTags = selectedTags.filter((item) => item !== label);
+      setSelectedTags(newTags);
+    } else {
+      newTags = [label];
+      setSelectedTags(newTags);
+    }
+  }
 
   const uploadImage = async (event) => {
     const files = event.target.files;
     const data = new FormData();
     data.append("file", files[0]);
     data.append("upload_preset", "meal_wheel");
-    try {
-      const uploadResponse = await fetch(
-        "https://api.cloudinary.com/v1_1/mealwheel/image/upload",
-        {
-          method: "POST",
-          body: data,
-        }
-      );
-      const file = await uploadResponse.json();
-      setImageUrl(file.secure_url);
-      notifySuccess("Bild hinzugefügt");
-    } catch (error) {
-      notifyError("Bild konnte nicht hinzugefügt werden");
-    }
+    const uploadResponse = await fetch(
+      "https://api.cloudinary.com/v1_1/mealwheel/image/upload",
+      {
+        method: "POST",
+        body: data,
+      }
+    );
+    const file = await uploadResponse.json();
+    setImageUrl(file.secure_url);
```

Comment:
this seems like it shouldn't be part of this US?

Reply:
Thank you very much for the comment. That was part of the [https://github.com/users/gcode-de/projects/1/views/1?pane=issue&itemId=55384387](userStory - addNewRecipe - imageAdding). The code was implemented and written one week ago. I cannot remember changing something in this function. So indeed not a part of this userStory.:)